website/docs/terminology.md

```diff
@@ -70,7 +70,8 @@ The control root is the Merkle hash of a set of [control ID]s.
 
 The control root plays a key role in enforcing zkVM version control. Each [control ID] identifies one [recursion program]; the control root identifies the list of allowed recursion programs. When the RISC-V circuit or recursion programs change, the control root is updated to reflect the new set of allowed functions.
 
-This design allows for updates to the [RISC-V circuit] without necessitating a new trusted setup for the [Groth16 circuit].
+This design allows for updates to the [RISC-V circuit] without necessitating a new trusted setup for the [Groth16 circuit]. <br/>
+See also: [Code reference for control root]
```

Comment:
Looks like you have changes from groth16 related content here. Was this intentional?

Reply:
hmm i can't recall whether I intentionally put these into the same PR or not; in retrospect I should have put them into separate PRs but these are sufficiently minor changes that I'm going to go ahead and land them.